pages/reference/api-support-matrix.en-US.mdx

```diff
@@ -80,7 +80,7 @@ import { Callout } from 'nextra-theme-docs';
 | Location                  | EU, US                                                                                                                           |
 | Codec                     | H264 + AAC                                                                                                                       |
 | H264 Transcoding Profiles | Baseline, Main, High                                                                                                             |
-| FPS                       | Supported tested: 24, 25, 30, 60 fps, uniform across all renditionsSupported untested: custom fps, uniform across all renditions |
+| FPS                       | Supported tested: 24, 25, 30, 60 fps, uniform across all renditions Supported untested: custom fps, uniform across all renditions |
```

Comment:
"uniform across all renditions Supported untested" 

☝️ is this a grammatical error?

Reply:
Yes

Reply:
Yes, I am guess there is suppose to be `Supported tested` and `Supported untested `line
![Screenshot 2023-06-20 at 9 22 54 AM](https://github.com/livepeer/docs/assets/53620329/876f0f30-0510-4f48-a7c0-7fb62c93964c)

Reply:
ah makes sense. Let's add a new line there, then good to merge.

Reply:
ok, just added a new line for them